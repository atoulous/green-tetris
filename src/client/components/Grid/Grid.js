import React from 'react';
import Row from '../Row/Row';

import './Grid.scss';

// todo add props validation
const Grid = ({ grid, move, additionalClasses, rowAdditionalClasses }) => {
  if (!rowAdditionalClasses) rowAdditionalClasses = [];
  if (!additionalClasses) additionalClasses = [];
  const classNames = `grid ${additionalClasses.join(' ')}`;

  return (
    <div className={classNames} onKeyDown={move} tabIndex="0">
      {grid.map((e, i) => <Row key={i} row={e} additionalClasses={rowAdditionalClasses} />)}
    </div>
  );
};

export default Grid;
