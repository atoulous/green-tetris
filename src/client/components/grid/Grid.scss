.grid {
  width: 100%;
  height: 100%;
  display: flex;
  flex-direction: column;
}


.grid .row {
  display: flex;
  flex-direction: row;
  flex: 1;
}

.grid .row .cell {
  border: solid gray 1px;
  display: flex;
  border-radius: 0;
  margin: 0;
  flex: 1;
}
