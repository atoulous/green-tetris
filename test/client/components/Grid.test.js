import React from 'react';
import renderer from 'react-test-renderer';

import Grid from '../../../src/client/components/Grid/Grid';

it('grid test', () => {
  const tree = renderer
    .create(<Grid grid={
      [
        [{ fill: false, color: 'grey', isDestroyable: true },
          { fill: false, color: 'grey', isDestroyable: true },
          { fill: false, color: 'blue', isDestroyable: true },
          { fill: false, color: 'grey', isDestroyable: true },
          { fill: false, color: 'grey', isDestroyable: true },
          { fill: false, color: 'grey', isDestroyable: true },
          { fill: false, color: 'grey', isDestroyable: true },
          { fill: false, color: 'grey', isDestroyable: true },
          { fill: false, color: 'grey', isDestroyable: true },
          { fill: false, color: 'grey', isDestroyable: true },
          { fill: false, color: 'grey', isDestroyable: true },
          { fill: false, color: 'grey', isDestroyable: true },
          { fill: false, color: 'grey', isDestroyable: true },
          { fill: false, color: 'grey', isDestroyable: true }],
          [{ fill: false, color: 'blue', isDestroyable: true },
            { fill: false, color: 'blue', isDestroyable: true },
            { fill: false, color: 'blue', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true }],
          [{ fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true }],
          [{ fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true }],
          [{ fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true }],
          [{ fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true }],
          [{ fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true }],
          [{ fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true }],
          [{ fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true }],
          [{ fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true }],
          [{ fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true }],
          [{ fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true }],
          [{ fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true }],
          [{ fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true }],
          [{ fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true }],
          [{ fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true }],
          [{ fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true }],
          [{ fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true }],
          [{ fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: true, color: 'blue', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true }],
          [{ fill: true, color: 'blue', isDestroyable: true },
            { fill: true, color: 'blue', isDestroyable: true },
            { fill: true, color: 'blue', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true },
            { fill: false, color: 'grey', isDestroyable: true }]
      ]
    }
    />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

