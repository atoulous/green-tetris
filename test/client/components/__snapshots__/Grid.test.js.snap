// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`grid test 1`] = `
<div
  className="grid "
  tabIndex="0"
>
  <div
    className="row"
  >
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
  </div>
  <div
    className="row"
  >
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
  </div>
  <div
    className="row"
  >
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
  </div>
  <div
    className="row"
  >
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
  </div>
  <div
    className="row"
  >
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
  </div>
  <div
    className="row"
  >
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
  </div>
  <div
    className="row"
  >
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
  </div>
  <div
    className="row"
  >
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
  </div>
  <div
    className="row"
  >
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
  </div>
  <div
    className="row"
  >
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
  </div>
  <div
    className="row"
  >
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
  </div>
  <div
    className="row"
  >
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
  </div>
  <div
    className="row"
  >
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
  </div>
  <div
    className="row"
  >
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
  </div>
  <div
    className="row"
  >
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
  </div>
  <div
    className="row"
  >
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
  </div>
  <div
    className="row"
  >
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
  </div>
  <div
    className="row"
  >
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
  </div>
  <div
    className="row"
  >
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={
        Object {
          "backgroundColor": "blue",
        }
      }
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
  </div>
  <div
    className="row"
  >
    <div
      className="cell"
      style={
        Object {
          "backgroundColor": "blue",
        }
      }
    />
    <div
      className="cell"
      style={
        Object {
          "backgroundColor": "blue",
        }
      }
    />
    <div
      className="cell"
      style={
        Object {
          "backgroundColor": "blue",
        }
      }
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
    <div
      className="cell"
      style={Object {}}
    />
  </div>
</div>
`;
